tests/conformance/utils/utils.go

```diff
@@ -134,3 +134,13 @@ func NewStringSet(values ...string) map[string]struct{} {
 
 	return set
 }
+
+func Contains[V comparable](arr []V, str V) bool {
```

Comment:
adding comment here ... so that file is not missed in the deluge of certification go.mod changes :)

Reply:
Can you use slices.Contains instead of a custom method? https://pkg.go.dev/golang.org/x/exp/slices#Contains